app/views/admin/shared/_logo_upload.html.erb

```diff
@@ -0,0 +1,12 @@
+ <div class="crayons-field mt-6 align-left" data-controller="logo-upload">
+  <%= label_tag :logo, class: "crayons-field__label" do %>
+    Logo
+    <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
+    <p id="logo-subtitle" class="crayons-field__description">You can upload a PNG or JPEG image.</p>
+  <% end %>
+  <div class="flex flex-1 gap-4">
+    <%= file_field_tag :logo, accept: allowed_types.to_s, data: { "max-file-size-mb": max_file_size.to_s, action: "change->logo-upload#previewLogo" }, aria: { describedby: "logo-subtitle" } %>
```

Comment:
refactor (blocking) -  moving the `aria-describedby` here will associate the description with the input as expected

```suggestion
    <%= file_field_tag :logo, accept: allowed_types.to_s, aria_describedby:"logo-subtitle", data: { "max-file-size-mb": max_file_size.to_s, action: "change->logo-upload#previewLogo" }, aria: { describedby: "logo-subtitle" } %>
```